db/migrate/20200312144232_rename_value_on_pr_state.rb

```diff
@@ -1,13 +0,0 @@
-class RenameValueOnPrState < ActiveRecord::Migration[6.0]
```

Comment:
Removing a migration? Why?

Reply:
Is going to be reverted. The structure is changed so when it gets on Heroku it will execute the migrations again and it will revert that one.